src/7-to-8/major-changes/continuing-c7-c8.rst

```diff
@@ -0,0 +1,41 @@
+.. _compat_continuing_c7_with_c8:
+
+Continuing a Cylc 7 Workflow with Cylc 8
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. admonition:: Does This Change Affect Me?
+   :class: tip
+
+   Read this if you have a partially complete Cylc 7 workflow that you want to
+   continue, rather than start from scratch, with Cylc 8. Some cycling
+   workflows, for example, may need to run expensive "cold start" tasks
+   and incur a multi-cycle spin-up if started from scratch.
+
+.. warning::
+
+   Cylc 8 cannot restart a Cylc 7 workflow in-place, and continuing in a new
+   run directory involves some careful set up (below). So, **if possible you
+   should complete the run with Cylc 7**.
+
+
+To continue a Cylc 7 workflow with Cylc 8:
+
+1. Stop the Cylc 7 workflow at an convenient place
+
+   - Typically the end of a cycle point, to simplify the continuation
+2. :ref:`Install <MajorChangesInstall>` a new instance of the workflow from
+   source, with Cylc 8
+
+   - Adapt file paths to the new run directory structure, in workflow and task
+     configurations
+   - Note Cylc 8 does :ref:`remote file installation <728.remote-install>`
+     when a job is first submitted to a platform
+3. Copy runtime files needed by upcoming tasks from the old to the new run
+   directory
+
+   - This could include external files installed by initial tasks at runtime
+   - Note different files could be present on different job platforms
+4. Start the new Cylc 8 run at the appropriate cycle point or task(s)
+
+   - To avoid re-running initial tasks, don't reset the :term:`initial
+     cycle point` to the new :term:`start point <start cycle point>`
```

Comment:
I'm not sure what the bullet point means. Perhaps it should just say "use the `--startcp`" option?

Reply:
I mean, don't change `initial cycle point` (in the `flow.cylc` or via the command line) to the start cycle of the Cylc 8 run, because that would result in a new cold start (i.e., from the beginning of the graph, with any attendant special behaviour in early cycles) that we're presumably trying to avoid with this set up procedure.

I'll expand on that a bit...